pages/implementation/start-custom-destination.mdx

```diff
@@ -77,6 +77,10 @@ The callback methods mirror the actions you set up for given event in your track
   Avo will log page view in your analytics tool when this event is sent. Usually set
   on events that are sent on screen/page navigation.
 
+- Update Group info
```

Comment:
```suggestion
- Update Groups
```

Reply:
Do we want to add something about `Log Event` for adding groups there?

Reply:
removed this section completely in https://github.com/avohq/docs/pull/216